internal/node_install.bzl

```diff
@@ -19,6 +19,8 @@ We then create a repository @nodejs which provides the
 node binary to other rules.
 """
 
+load(":node_labels.bzl", "get_node_label")
```

Comment:
can you eliminate the uses of `get_executables` at the same time?

Reply:
👍 will do